axelor-business-project/src/main/java/com/axelor/apps/businessproject/service/PurchaseOrderLineProjectService.java

```diff
@@ -19,13 +19,18 @@
 package com.axelor.apps.businessproject.service;
 
 import com.axelor.apps.project.db.Project;
+import com.axelor.apps.project.db.ProjectTask;
 import com.axelor.apps.purchase.db.PurchaseOrderLine;
+import com.google.inject.persist.Transactional;
 import java.util.List;
 
 public interface PurchaseOrderLineProjectService {
 
   public void setProject(List<Long> purchaseOrderLineIds, Project project);
 
+  @Transactional
```

Comment:
please remove this transactional annotation in the interface, as it will be ignored.